roboteam_ai/test/ControlTests/ControlUtilsTest.cpp

```diff
@@ -45,6 +45,37 @@ TEST(ControlUtils, velocityLimiter) {
     }
 }
 
+TEST(ControlUtils, accelerationLimiter) {
+    rtt::Vector2 testVel;
+    double prevTestVel;
+    for (int i = 1; i < 200; i ++) {
+        testVel = Vector2(5*cos(i/100.0*M_PI), 3*sin(i/30.0*M_PI));
+        prevTestVel = Vector2(5*cos((i-1)/100.0*M_PI), 3*sin((i-1)/30.0*M_PI)).length();
+        EXPECT_LE(cr::ControlUtils::accelerationLimiter(testVel, rtt::ai::Constants::MAX_ACC_UPPER(), prevTestVel).length(),
```

Comment:
use using statements so  something like using c=rtt::ai::constants so this is actually kind of readable.

Reply:
this goes for all the tests